packages/ketcher-react/src/script/render/restruct/resimpleobject.js

```diff
@@ -210,17 +210,17 @@ ReSimpleObject.prototype.highlightPath = function (render) {
         render.paper.ellipse(
           tfx(point[0].x + rx),
           tfx(point[0].y + ry),
-          tfx(Math.abs(rx) + s / 8),
-          tfx(Math.abs(ry) + s / 8)
+          tfx(Math.abs(rx) + scale / 8),
```

Comment:
Are you shure about `scale` and not `scaleFactor`? I guess here is some mistake

Reply:
thank you, fixed